cmd/apply.go

```diff
@@ -54,7 +54,8 @@ func RunApply(cmd *cobra.Command, args []string) {
 			secret, err = c.UpdateSecret(secret)
 		} else {
 			cmd.Println(secretAskConfirmationMessage)
-			err = utils.Ask(secret.Metadata.Name)
+			cmd.InOrStdin()
```

Comment:
Why do we need this extra call of `cmd.InOrStdin()`?

Reply:
We do not, I probably pasted it in the wrong place :)